### PR TITLE
Look 159

### PR DESCRIPTION
**Pull request by Simarglok**

### CODE REVIEW

File: models/marts/sales/orders_flow_details.sql [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+	max(case type when 20 then sf_guard_user_id end) as delivery_stopped_by_id,
+	max(case type when 21 then created_at end) as rejected_at,
+	max(case type when 21 then sf_guard_user_name end) as rejected_by,
+	max(case type when 21 then sf_guard_user_id end) as rejected_by_id,
+	max(case type when 39 then created_at end) as delivery_accepted_at,
+	max(case type when 39 then sf_guard_user_name end) as delivery_accepted_by,
+	max(case type when 39 then sf_guard_user_id end) as delivery_accepted_by_id,
+	max(case type when 43 then created_at end) as exchanged_at,
+	max(case type when 43 then sf_guard_user_name end) as exchanged_by,
+	max(case type when 43 then sf_guard_user_id end) as exchanged_by_id
```
Comment (iplotnikov-telemedicine) [inline models/marts/sales/orders_flow_details.sql:98]: There is a concise way to write this kind of thing in dbt with a loop: https://docs.getdbt.com/docs/get-started/learning-more/using-jinja#use-a-for-loop-in-models-for-repeated-sql
You can try it if you have enough time to figure it out

**Review by iplotnikov-telemedicine — COMMENTED** [review]
Merging at your descretion

File: models/marts/sales/orders_flow_details.sql (hunk 1 above)
Comment (Simarglok) [reply models/marts/sales/orders_flow_details.sql]: I updated this file with your recommendations. Pls check it again.

**Comment by iplotnikov-telemedicine** [comment]
@Simarglok nice job

**Pull request merged**